### PR TITLE
Automatic prod deploy

### PR DESCRIPTION
**Pull request by StanislavBreadless**

### CODE REVIEW

File: .github/workflows/firebase-hosting-merge.yml [hunk 1]
```diff
@@ -8,15 +8,35 @@ name: Deploy to Firebase Hosting on merge
...
+        
+      
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
+          firebaseServiceAccount: '${{ secrets.FIREBASE_SERVICE_ACCOUNT_FORCED_EXIT_MAINNET }}'
+          channelId: live
+          projectId: zksync-forced-withdrawal
+        env:
+          FIREBASE_CLI_PREVIEWS: hostingchannels
```
Comment (popzxc) [inline .github/workflows/firebase-hosting-merge.yml:42]: (ಠ⌣ಠ)

File: nuxt.config.js [hunk 2]
```diff
@@ -1,6 +1,13 @@
-require('dotenv').config()
+require('dotenv').config();
 
-const pageTitle = 'zkSync Alternative Withdrawal'
+const pageTitle = 'zkSync Alternative Withdrawal from L2 back to mainnet';
+const pageDescription = 'Easily move your funds to L1 without interaction with L2.'
+const pageKeywords = "zksync withdrawal, alternative withdrawal, bulk withdrawal, zkSync, Matter Labs, rollup, ZK rollup, zero confirmation, ZKP, zero-knowledge proofs," +
```
Comment (popzxc) [inline nuxt.config.js:5]: Why not `array.join`?

**Review by popzxc — APPROVED** [review]
PR contents look OK to me; however, I don't know whether it serves its purpose :)

File: nuxt.config.js [hunk 3]
```diff
@@ -68,17 +99,30 @@ export default {
    ** Nuxt.js modules
    */
   modules: [
-    '@nuxtjs/dotenv',
     '@nuxtjs/pwa',
     '@nuxtjs/axios',
     '@inkline/nuxt',
     'vue-scrollto/nuxt',
     '@nuxtjs/style-resources',
```
Comment (keriat) [inline nuxt.config.js:76]: -This package should be added to the ```buildModules```

File: nuxt.config.js [hunk 4]
```diff
@@ -68,17 +99,30 @@ export default {
    ** Nuxt.js modules
    */
   modules: [
-    '@nuxtjs/dotenv',
     '@nuxtjs/pwa',
```
Comment (keriat) [inline nuxt.config.js:72]: This one is safe to drop

**Review by keriat — CHANGES_REQUESTED** [review]
The file nuxt.config should be saved as a .ts

Also please review the changes in eslint / prettified / style it.
Update the packages and and strict typing.
All ok it can be taken either from the zkWallet (the only available PR) or from the [zksync-dApp-withdrawal](https://github.com/matter-labs/zksync-dapp-withdrawal/pulls)

Also make sure to:
- [ ] Set up tsconfig.json
- [ ] update packages and add types
- [ ] replace the favicon located inside the /src/static/icon.png
- [ ] add the sentry / gtag services especially for it
- [ ] move all configuration scss files to the assets/style/vars and add preprocessing onBuild into nuxt.config
- [ ] add newly built cli-...env... to handle dev/prod environments and publish the package
- [ ] Apply strict types to the /src/stores/ using #typed-vuex package
- [ ] add links to the products and documentation of zkSync into the READMe.md / package.json

Helpful info:
- [Typescript.nuxt.org](Typescript.nuxt.org)
- [Typed-vuex](https://typed-vuex.roe.dev)
- [complete nuxt typescript guide](https://al-un.github.io/learn-nuxt-ts/)

**Comment by StanislavBreadless** [comment]
@keriat 

This PR is about automatic prod deployment + some small cosmetic changes. 

I propose that I do the following:

- [ ] replace the favicon located inside the /src/static/icon.png
- [ ] add links to the products and documentation of zkSync into the READMe.md / package.json
- [ ] Fix the parts that you commented about

The rest of the changes requested are very needed for the tool and the ability to maintain it in the long-term, but out-of-scope of this PR. I can create a task for them.

**Review by keriat — COMMENTED** [review]
> @keriat
> 
> This PR is about automatic prod deployment + some small cosmetic changes.
> 
> I propose that I do the following:
> 
> * [ ]  replace the favicon located inside the /src/static/icon.png
> * [ ]  add links to the products and documentation of zkSync into the READMe.md / package.json
> * [ ]  Fix the parts that you commented about
> 
> The rest of the changes requested are very needed for the tool and the ability to maintain it in the long-term, but out-of-scope of this PR. I can create a task for them.

Let's add here two more things:

* [ ] Update Nuxt (package.json, nuxt.config.ts and relevant lint-tools)
* [ ] Replace Header / Footer / Logginin with zksync-dapp-withdrawal

**Comment by keriat** [comment]
> > @keriat
> 
> > 
> 
> > This PR is about automatic prod deployment + some small cosmetic changes.
> 
> > 
> 
> > I propose that I do the following:
> 
> > 
> 
> > * [ ]  replace the favicon located inside the /src/static/icon.png
> 
> > * [ ]  add links to the products and documentation of zkSync into the READMe.md / package.json
> 
> > * [ ]  Fix the parts that you commented about
> 
> > 
> 
> > The rest of the changes requested are very needed for the tool and the ability to maintain it in the long-term, but out-of-scope of this PR. I can create a task for them.
> 
> 
> 
> Let's add here two more things:
> 
> 
> 
> * [ ] Update Nuxt (package.json, nuxt.config.ts and relevant lint-tools)
> 
> * [ ] Replace Header / Footer / Logginin with zksync-dapp-withdrawal

We've decided to push this PR and postpone changes.
@StanislavBreadless Please create some task w/t the things that require further effort.

**Review by keriat — APPROVED** [review]

**Pull request merged**